server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -740,6 +791,33 @@ writing keys [[B0x033E243034396234..[39]]: org.infinispan.util.concurrent.Timeou
 +
 Those exceptions are not necessarily an issue. They may happen anytime when concurrent edit of same entity is triggered on both DCs. This can be the often case in a deployment. Usually the {project_name} server is notified about the failed operation and will retry it, so from the user's point of view, there is usually not any issue.
 
+* If there are exceptions during startup of {project_name} server like this:
++
+```
+16:44:18,321 WARN  [org.infinispan.client.hotrod.impl.protocol.Codec21] (ServerService Thread Pool -- 55) ISPN004005: Error received from the server: java.lang.SecurityException: ISPN000287: Unauthorized access: subject 'Subject with principal(s): []' lacks 'READ' permission
+ ...
+```
++
+These log entries come from automatic detection by {project_name} of whether authentication is required on {jdgserver_name} and mean
+that the authentication is indeed necessary. Now either the server starts successfully and you can safely ignore these. If, on the
+other hand, the server fails to start, ensure that {jdgserver_name} has been configured properly for authentication as described in
+<<jdgsetup>>. To prevent this log entry to occur, you can force authentication by setting `remoteStoreSecurityEnabled` property to
+`true` in `spi=connectionsInfinispan/provider=default` configuration:
++
+```xml
+        <subsystem xmlns="urn:jboss:domain:keycloak-server:1.1">
+            ...
+            <spi name="connectionsInfinispan">
+                ...
+                <provider name="default" enabled="true">
+                    <properties>
+                        ...
+                        <property name="remoteStoreSecurityEnabled" value="true"/>
+                    </properties>
+                </provider>
+            </spi>
+```
+
 * If you try to authenticate with {project_name} to your application, but it failed with the infinite number 
 of redirects in your browser and you see the errors like this in the {project_name} server log:
 +
```

Comment:
* If you try to authenticate with {project_name} to your application, but authentication fails with an infinite number 
 of redirects in your browser and you see the errors like this in the {project_name} server log: